src/generate_worker_metadata.py

```diff
@@ -34,7 +34,7 @@ def generate_worker_metadata(worker_subcls: Type[BaseWorker], package_name: str)
         )
     )
     validate = fastjsonschema.compile(worker_schema)
-    validate(worker_schema)
+    validate(worker_metadata)
```

Comment:
I think `validate_view_content` is now expecting `getattr(metadata_schemas, f"{variety}_schema")` -> `dict`

I can update that so its more general, but just flagging since i think this would break it